cmd/kubeadm/app/phases/kubelet/kubelet.go

```
@@ -45,6 +68,20 @@ func TryStartKubelet() {
 	}
 }
 
+// PrintKubeletErrorHelpScreen prints help text on kubelet errors.
+func PrintKubeletErrorHelpScreen(outputWriter io.Writer, criSocket string, controlPlane bool) {
+	context := struct {
+		Socket string
+	}{
+		Socket: criSocket,
+	}
+
+	fmt.Fprintln(outputWriter, kubeletFailMsg)
+	if controlPlane {
+		_ = controlPlaneFailTempl.Execute(outputWriter, context)
```

We didn't print handle the error(probably print it) before. Not sure if we should do that.

---

In other parts of kubeadm, calls to the Execute method similarly do not return errors, so I chose to ignore the error here.